CLM-8307 ContentHeader Component (#206)
thoughtindustries/helium: packages/content/README.md

  });
}
```

> ## Content Header

Reviewer: Would make the below changes in https://github.com/thoughtindustries/ti-docs/pull/18 as well.

> The `Content Header` component is used to add a title, description, rating and image to a course quickly. This is done by passing props to a component that is already created, rather than creating a new one from scratch.

Reviewer: ```suggestion
The `Content Header` component is used to display the `title`, `description`, `rating`, `ratingsCount` and image of a Course or Learning Path.
```

> ### Example code

Reviewer: ```suggestion
## Example code
```

```tsx
import { ContentHeader } from '@thoughtindustries/content';

export function MyComponent() {
  // ...

  return (
    <ContentHeader contentKind={contentKind} slug={courseSlug} showStars={true} showImage={true} />
  );
}
```

## Props

| Name        | Required | Type                 | Description                                                    |
| ----------- | -------- | -------------------- | -------------------------------------------------------------- |

> | contentKind | Yes      | <code>string</code>  | The contentKind of the Content. LearningPath, Course elements. |

Reviewer: ```suggestion
| contentKind | Yes      | <code>string</code>  | The contentKind of the Content, e.g., `learningPath`, `course`. |
```

> | slug        | Yes      | <code>slug</code>    | Content Slug cart.                                             |

Reviewer: ```suggestion
| slug        | Yes      | <code>slug</code>    | The slug of the content that should be displayed.                                             |
```

> | showStars   | No       | <code>boolean</code> | If false, hide the stars. If true, show the stars              |

Reviewer: ```suggestion
| showStars   | No       | <code>boolean</code> | Controls whether a course's rating is displayed, shown in stars.              |
```

> | showImage   | No       | <code>boolean</code> | If false, hide the image. If true, show the image              |

Reviewer: ```suggestion
| showImage   | No       | <code>boolean</code> | Controls whether a piece of content's Detail Image is displayed.              |
```
